tests/test_read_omb_csv.py

```diff
@@ -0,0 +1,23 @@
+import pandas as pd
+from trajan.readers.omb import read_omb_csv
+
+
+def test_read_csv_omb_default_waves(test_data):
+    path_to_test_data = test_data / 'csv' / 'omb1.csv'
+    ds = read_omb_csv(path_to_test_data)
+
+    # 1668207637
+    assert ds.attrs['time_coverage_start'] == '2022-11-12T00:00:37+00:00'
+    assert ds.attrs['time_coverage_end'] == '2022-11-12T02:30:27+00:00'
+
+
+def test_read_csv_omb_modified_waves(test_data):
+    path_to_test_data = test_data / 'csv' / 'omb2.csv'
+
+    dict_wave_packet_params = {
+        "_BD_YWAVE_PACKET_MIN_BIN": 9,
```

Comment:
Any reason not to keep these as regular arguments to the function? Then you'd catch syntaxerrors in the argument.

Reply:
The idea is that this is only for advanced users who know what they do, and if some advanced users want to implement more "deviations from the defaults", I think providing a single dict with modified decoder params will be more flexible. I think providing all this data as arguments would result in lots of arguments, different combinations needed, etc, so would be complex too. Maybe the "best" solution would be to define a dataclass DecodeOptions containing all these options, but this would be quite a bit of refactoring, not sure I am able to find time for doing this.

So I think that given how little this will be used, and who will use it, that this is an ok tradeoff for now?

Reply:
Ok, sounds good.